### ISSUE TITLE
How to add Exhibitors listing to my event 

### ISSUE THREAD

**Issue by iamdevvalecha** [fossasia/support.eventyay.com#230]

**Comment by piyushjasaiwal** [comment]
i would like to work on this issue, can you please help me how to get started on this as there is no description.

**Comment by iamdevvalecha** [comment]
@piyushjasaiwal you could create a markdown file and write the article for this issue. you could support page meeting on thursday 
8 pm IST https://meet.jit.si/eventyay.

**Comment by piyushjasaiwal** [comment]
ok sir thank you now i understand how to accomplish this

**Comment by piyushjasaiwal** [comment]
thank you @iamdevvalecha sir

**Comment by piyushjasaiwal** [comment]
sir in which folder i have to make a .md file for exhibitor listing.

**Comment by iamdevvalecha** [comment]
make a new folder Sponsoring and Exhibition then make .md file and write the article.

**Comment by piyushjasaiwal** [comment]
one more doubt what do you mean by an exhibitor 
are exhibtor or speaker same

**Comment by iamdevvalecha** [comment]
no,they both are different exhibitor are those who present work in exhibition

**Comment by piyushjasaiwal** [comment]
exhibitor is a name for sponsor i guess.

**Comment by iamdevvalecha** [comment]
Nope

**Comment by piyushjasaiwal** [comment]
is exhibitor the organization that set up the event
or the persons which are setting the event

**Comment by iamdevvalecha** [comment]
@piyushjasaiwal https://meet.jit.si/eventyay
join this i will clear your doubts
